specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json

```diff
@@ -0,0 +1,10921 @@
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "sshPublicKeyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the SSH public key."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SshPublicKeyResource"
+            },
+            "description": "Parameters supplied to create the SSH public key."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SshPublicKeyResource"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/SshPublicKeyResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a new SSH public key resource.": {
+            "$ref": "./examples/CreateAnSshPublicKey.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "SshPublicKeys"
+        ],
+        "operationId": "SshPublicKeys_Update",
+        "description": "Updates a new SSH public key resource.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "sshPublicKeyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the SSH public key."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SshPublicKeyUpdateResource"
+            },
+            "description": "Parameters supplied to update the SSH public key."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SshPublicKeyResource"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SshPublicKeys"
+        ],
+        "operationId": "SshPublicKeys_Delete",
```

Comment:
long-running?

Reply:
[5:25 PM] Nathan Kuchta
    No, they are not long running and return immediately